### PR TITLE
[MOSIP-32144] code coverage of authentication-service

### PR DESCRIPTION
**Pull request by tarique-azeez**
## Summary by CodeRabbit

* **Tests**
  * Expanded credential-store tests: added concurrent pool init checks, new exception-handling paths, renamed/added test cases, and save-on-failure verifications.
  * Added comprehensive auth-config test suite for env-driven auth toggles and many edge cases.
  * Broadened identity-key-binding and KYC tests with reflected private-path coverage, token/metadata scenarios, and error paths.
  * Added targeted tests for address-claim, verifiable-credential error flows, and improved test scaffolding/imports.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: authentication/authentication-internal-service/src/test/java/io/mosip/authentication/internal/service/batch/CredentialStoreTaskletTest.java [hunk 1]
```diff
@@ -90,16 +96,124 @@ public void testExecute_withValidEvents_shouldProcessAndSave() throws Exception
...
                 .thenThrow(new RuntimeException("Generic"));
 
         RepeatStatus status = tasklet.execute(contribution, chunkContext);
 
         assertEquals(RepeatStatus.FINISHED, status);
+        verify(credentialEventRepo).saveAll(events);
+    }
+
+    @Test
+    public void testExecute_withIdAuthenticationBusinessException_shouldContinue() throws Exception {
```
Comment (JanardhanBS-SyncByte) [inline authentication/authentication-internal-service/src/test/java/io/mosip/authentication/internal/service/batch/CredentialStoreTaskletTest.java]: java camel case change function naming convention .. ex: javaNotFollowed()

**Review by JanardhanBS-SyncByte — CHANGES_REQUESTED** [review]

File: authentication/authentication-internal-service/src/test/java/io/mosip/authentication/internal/service/batch/CredentialStoreTaskletTest.java (hunk 1 above)
Comment (JanardhanBS-SyncByte) [reply authentication/authentication-internal-service/src/test/java/io/mosip/authentication/internal/service/batch/CredentialStoreTaskletTest.java]: testExecute_withIdAuthenticationBusinessException_shouldContinue() should become testExecuteWithIdAuthenticationBusinessExceptionShouldContinue()

Comment (tarique-azeez) [reply authentication/authentication-internal-service/src/test/java/io/mosip/authentication/internal/service/batch/CredentialStoreTaskletTest.java]: done for all the methods

File: authentication/authentication-internal-service/src/test/java/io/mosip/authentication/internal/service/batch/CredentialStoreTaskletTest.java [hunk 2]
```diff
@@ -62,8 +62,11 @@ public class CredentialStoreTaskletTest {
     public void setUp() {
         MockitoAnnotations.initMocks(this);
         ReflectionTestUtils.setField(tasklet, "threadCount", 4);
+        ReflectionTestUtils.setField(tasklet, "pageSize", 100);
+        when(securityManager.getUser()).thenReturn("test-user");
         tasklet.init();
     }
+
     @Test
     public void testExecute_withEmptyList_shouldNotCallSaveAll() throws Exception {
```
Comment (JanardhanBS-SyncByte) [inline authentication/authentication-internal-service/src/test/java/io/mosip/authentication/internal/service/batch/CredentialStoreTaskletTest.java]: java camel case

File: authentication/authentication-internal-service/src/test/java/io/mosip/authentication/internal/service/batch/CredentialStoreTaskletTest.java [hunk 3]
```diff
@@ -90,16 +93,48 @@ public void testExecute_withValidEvents_shouldProcessAndSave() throws Exception
         verify(credentialStoreService).storeIdentityEntity(identity);
         verify(credentialEventRepo).saveAll(events);
     }
+
     @Test
     public void testExecute_withGenericException_shouldContinue() throws Exception {
```
Comment (JanardhanBS-SyncByte) [inline authentication/authentication-internal-service/src/test/java/io/mosip/authentication/internal/service/batch/CredentialStoreTaskletTest.java]: java camel case

**Review by JanardhanBS-SyncByte — CHANGES_REQUESTED** [review]
check for java camel case and coderabbit reviews

**Review by JanardhanBS-SyncByte — APPROVED** [review]

**Review by mahammedtaheer — APPROVED** [review]
